### PR TITLE
Add .NET SDK examples to guides

### PR DESCRIPTION
**Pull request by adityaoberai**
## What does this PR do?

Adds .NET SDK examples to guides

## Test Plan

- Visit the https://github.com/appwrite/homepage repo
- Update `appwrite/docs` field to `dev-add-dotnet-examples` in `composer.json`
- Run development setup as stated in the readme on homepage repo

## Related PRs and Issues

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by gewenyu99 — APPROVED** [review]
LGTM

File: app/views/docs/getting-started-for-server.phtml [hunk 1]
```diff
@@ -521,6 +551,22 @@ let user = try await users.create(
...
+using Appwrite.Models;
+
+var users = new Users(client);
+
+var user = await users.Create(
+    userId: ID.Unique(),
+    email: "email@example.com",
+    phone: null,
+    password: "password");
+</code></pre>
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-server.phtml:567]: Ditto, unnecessary nextline

File: app/views/docs/getting-started-for-server.phtml [hunk 2]
```diff
@@ -711,6 +757,28 @@ let user = try await users.create(
...
+  .SetKey("cd868db89");                         // Your secret API key
+
+var users = new Users(client);
+
+var user = await users.Create(
+    userId: ID.Unique(),
+    email: "email@example.com",
+    phone: null,
+    password: "password");
+</code></pre>
```
Comment (gewenyu99) [inline app/views/docs/getting-started-for-server.phtml:779]: Ditto, delete next line

**Comment by adityaoberai** [comment]
@christyjacob4 let's merge this?

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**